SPEC.md

```diff
@@ -7951,18 +7951,26 @@ And `/local/fs/tmp/array.txt` would contain:
 ### `read_tsv`
 
 ```
-Array[Array[String]] read_tsv(File)
+Array[Array[String]] read_tsv(File), Array[Object] read_tsv(File, true), Array[Object] read_tsv(File, Boolean, Array[String])
```

Comment:
Please put each signature on a different line, like how we've done it for the other overloaded functions

Reply:
Fixed